theme.toml

```diff
@@ -60,6 +60,10 @@ error = "apricot"
 info = "delta"
 hint = "silver"
 
+line_added = "#35BF86"
+line_removed = "#F22C86"
+line_modified = "#6F44F0"
```

Comment:
I think you probably want to add this for other themes as well.

Reply:
Check